bundles/org.openhab.binding.ekey/README.md

```diff
@@ -14,12 +14,13 @@ This binding only supports one thing type:
 
 The binding uses the following configuration parameters.
 
-| Parameter | Description                                                    |
-|-----------|----------------------------------------------------------------|
-| ipAddress | IPv4 address of the eKey udp converter.  A static IP address is recommended.|
-| port      | The port as configured during the UDP Converter configuration.  e.g. 56000 (Binding default)     |
-| protocol  | Can be RARE, MULTI or HOME depending on what the system supports. Binding defaults to RARE  |
-| delimiter | The delimiter is also defined on the ekey UDP converter - use the ekey configuration software to determine which delimiter is used or to change it.  Binding default is `_` (underscore)  |
+| Parameter | Description                                                                                                                                                                              |
+|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ipAddress | IPv4 address of the eKey udp converter.  A static IP address is recommended.                                                                                                             |
+| port      | The port as configured during the UDP Converter configuration.  e.g. 56000 (Binding default)                                                                                             |
+| protocol  | Can be RARE, MULTI or HOME depending on what the system supports. Binding defaults to RARE                                                                                               |
+| delimiter | The delimiter is also defined on the ekey UDP converter - use the ekey configuration software to determine which delimiter is used or to change it.  Binding default is `_` (underscore) |
+| natIp  | [Optional] IPv4 address of a received eKey udp packet. Can be different from the ipAddress when using NAT. (e.g. in Kubernetes)                                                          |

```

Comment:
As you tried to maintain a proper alignment (thank you), can you do it please  by adding some spaces after "natIp" to have the beginning of your second column properly aligned ?